From dfa22cc55fb64787334f0e9a6730043dc8f291ac Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 31 May 2024 10:55:59 +0200
Subject: [PATCH] Fix dependence on phonolammps

---
 atomistics/calculators/lammps/__init__.py | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/atomistics/calculators/lammps/__init__.py b/atomistics/calculators/lammps/__init__.py
index 54f7493..709e855 100644
--- a/atomistics/calculators/lammps/__init__.py
+++ b/atomistics/calculators/lammps/__init__.py
@@ -19,11 +19,14 @@
     from atomistics.calculators.lammps.phonon import (
         calc_molecular_dynamics_phonons_with_lammps,
     )
+    __all__ = [
+        calc_molecular_dynamics_phonons_with_lammps,
+    ]
 except ImportError:
-    pass
+    __all__ = []
 
 
-__all__ = [
+__all__ += [
     calc_molecular_dynamics_thermal_expansion_with_lammps,
     calc_molecular_dynamics_nph_with_lammps,
     calc_molecular_dynamics_npt_with_lammps,
@@ -36,5 +39,4 @@
     optimize_positions_with_lammps,
     get_potential_dataframe,
     get_potential_by_name,
-    calc_molecular_dynamics_phonons_with_lammps,
 ]
