From c3fb659aa603db709477346dac697869809dd7db Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 29 Sep 2025 18:20:18 +0200
Subject: [PATCH] fix scipy

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2aaebfa..4c5f103 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,7 +26,7 @@ classifiers = [
 dependencies = [
     "ase>=3.23.0,<=3.26.0",
     "numpy>=1.26.4,<=2.3.3",
-    "scipy>=1.11.2,<=1.15.2",
+    "scipy>=1.11.2,<=1.16.2",
     "spglib>=2.0.2,<=2.6.0",
 ]
 dynamic = ["version"]
